src/parser.cc

```diff
@@ -1944,17 +1944,17 @@ bool convertFile(const std::string &_filename, const std::string &_version,
 bool convertFile(const std::string &_filename, const std::string &_version,
                  const ParserConfig &_config, SDFPtr _sdf)
 {
-  sdf::Errors errors;
-  bool result = convertFile(_filename, _version, _config, _sdf, errors);
+  sdf::Errors errors = convertFile(_sdf, _filename, _version, _config);
   throwOrPrintErrors(errors);
-  return result;
+  return errors.empty();
 }
 
 /////////////////////////////////////////////////
-bool convertFile(const std::string &_filename, const std::string &_version,
-                 const ParserConfig &_config, SDFPtr _sdf,
-                 sdf::Errors &_errors)
+sdf::Errors convertFile(SDFPtr _sdf, const std::string &_filename,
```

Comment:
@scpeters Wouldn't this break API?

Reply:
this is targeting your branch, which hasn't been merged to `sdf13` yet. It changes the proposed API. Comparing the branch to `sdf13`, nothing is broken: https://github.com/gazebosim/sdformat/compare/sdf13...scpeters/sdf_error_converter

Reply:
True, my bad.